src/operations/reweight.lisp

```diff
@@ -126,7 +134,7 @@
     (let ((rewinding-pong nil)
           (original-amount (message-pong-weight original-pong)))
       (flet ((payload-constructor ()
-               (make-message-soft-scan :weight 0 :repeat? t)))
+               (make-message-soft-scan :weight 0 :strategy :STAY)))
```

Comment:
it's common style to write this like `:strategy ':STAY` to emphasize that `:strategy` is a key identifier, consumed almost like macro fodder, whereas `':STAY` is actual data which is part of the computation. dunno how consistent we are about this to begin with, so may not be worth taking a stand about here.

Reply:
I'm happy to do this -- I just get a little mixed up with what I should and shouldn't quote sometimes, but I personally think quoting the keyword would be clearer actually (and therefore agree w/ you and the common style).